README.md

```diff
@@ -60,6 +60,14 @@ Additional Conda installation options available on the [gh-feedstock page](https
 
 For more information about the Webi installer see [its homepage](https://webinstall.dev/).
 
+#### Flox
+
+| Install:          | Upgrade:                |
+| ----------------- | ----------------------- |
+| `flox install gh` | `flox upgrade toplevel` |
+
+For more information about Flox, see [its homepage](https://flox.dev)
+
```

Comment:
I think moving this under the unoffical, community supported documentation makes more sense given this is support the Flox community is extending:

https://github.com/cli/cli/blob/89cbcfe7eb186ff4edbe10792d17bdc55b04f297/docs/install_linux.md?plain=1#L99-L101

Reply:
Something like:

```
### Flox

Flox users can install from the [official community nixpkgs](https://github.com/flox/nixpkgs).

For more information about Flox, see [its homepage](https://flox.dev)

```bash
# To install
flox install gh

# To upgrade
flox upgrade toplevel
```

Reply:
I think there's a bit of an issue with our taxonomy on the README. Like, we have an expanding MacOS section in the README, and a separate linux page.

However, homebrew is on linux, and nix and flox are on MacOS, so it's kind of weird? Maybe we just need to revisit our structure.

In the interest of getting this merged I've duplicated the information.